_includes/releases/v21.2/v21.2.7.md

```diff
@@ -0,0 +1,86 @@
+## v21.2.7
+
+Release Date: March 14, 2022
+
+{% include releases/release-downloads-docker-image.md release=include.release %}
+
+<h3 id="v21-2-7-security-updates">Security updates</h3>
+
+- When the `sql.telemetry.query_sampling.enabled` [cluster setting](../v21.2/cluster-settings.html) is enabled, SQL names and client IP addresses are no longer redacted in telemetry logs. [#77072][#77072]
+
+<h3 id="v21-2-7-enterprise-edition-changes">Enterprise edition changes</h3>
+
+- Currently executing schedules are cancelled immediately when the jobs scheduler is disabled.  Release justification: stability improvement. [#77313][#77313]
+
+<h3 id="v21-2-7-sql-language-changes">SQL language changes</h3>
+
+- When dropping a user that has default [privileges](../v21.2/security-reference/authorization.html#privileges), the error message now includes which database and schema the default privileges are defined in.  Additionally, a hint is now given to show exactly how to remove the default privileges. [#77142][#77142]
+- Previously, `crdb_internal.default_privileges` would incorrectly show default [privileges](../v21.2/security-reference/authorization.html#privileges) for databases where the default privilege was not actually defined. [#77304][#77304]
+
+<h3 id="v21-2-7-operational-changes">Operational changes</h3>
+
+- Operators who wish to access HTTP endpoints of a cluster through a proxy can now request specific node IDs through a `remote_node_id` query parameter or cookie with the value set to the node IDs they would like to proxy the connection to. [#76694][#76694]
+- Added the [cluster setting](../v21.2/cluster-settings.html) `bulkio.backup.resolve_destination_in_job.enabled`, which can be used to delay resolution of backup's destination until the job starts running. [#76816][#76816]
+
+<h3 id="v21-2-7-db-console-changes">DB Console changes</h3>
+
+- Open SQL Transactions and Active SQL Transactions are now downsampled using MAX instead of AVG and will more accurately reflect narrow spikes in transaction counts when looking and downsampled data. [#76688][#76688]
```

Comment:
```suggestion
- Open SQL Transactions and Active SQL Transactions are now downsampled using `MAX` instead of `AVG` and will more accurately reflect narrow spikes in transaction counts when looking and downsampled data. [#76688][#76688]
```